app/src/androidTest/java/com/github/palFinderTeam/palfinder/meetups/activities/MeetupListTest.kt

```diff
@@ -97,7 +98,7 @@ class MeetUpListTest {
                 description = "nous aimerions bien nous atteler à la cuisson de patates au beurre",
                 startDate = date2,
                 endDate = date1,
-                location = Location(0.0, 0.0),
+                location = Location(-122.0, 37.0),
```

Comment:
Do you need all events to take place at the same place?

Reply:
yes, because we only fetch around location, I found it easier to group them around the user location